### DISCUSSION TITLE
Question about packaging h5fuse.in and h5tools_test_utils

### DISCUSSION THREAD

**Discussion by opoplawski** [HDFGroup/hdf5#3791] [General]
I'm looking at test building the Fedora hdf5 with the current hdf5_14 branch.  I'm now getting the following new executables installed:
```
   /usr/bin/h5fuse.sh
   /usr/bin/h5tools_test_utils
```
First, is this expected?  If so, are the runtime or development related?  Thanks

**Comment by jhendersonHDF** [comment]
Hi @opoplawski,

`h5fuse.sh` is a "tool" script related to the Subfiling VFD that takes files written by that VFD and fuses them back into a valid HDF5 file that can be read as normal. You can read more about the VFD and the tool at https://github.com/HDFGroup/hdf5doc/blob/master/RFCs/HDF5_Library/VFD_Subfiling/user_guide/HDF5_Subfiling_VFD_User_s_Guide.pdf. We decided to always distribute that script in case someone wishes to be able to fuse Subfiling VFD files back together without needing to enable and build the VFD itself.

As for `h5tools_test_utils`, this is a test program for testing some utility functions mostly related to the ROS3 VFD. I would not expect that test program to be installed and I believe that it was mistakenly added to `bin_PROGRAMS` rather than `noinst_PROGRAMS`. Our CMake code doesn't install the test program, so I'll fix this. Thanks for the catch!

**Comment by opoplawski** [comment]
@jhendersonHDF Thanks for the info.  One comment - I would suggest dropping the `.sh` suffix unless you expect there to be separate `h5fuse` binary in addition to the script.
